easybuild/easyconfigs/m/MapSplice/MapSplice-2.2.1_fix-build-gcc.patch

```diff
@@ -0,0 +1,41 @@
+* fix build error with gcc >= 4.9: redeclaration of ... may not have default arguments
```

Comment:
@SethosII please include author (+ affiliation) or reference for this patch

Reply:
@boegel I changed this.